src/conftests/sdl2_mixer.cpp

```diff
@@ -13,7 +13,6 @@
 */
 
 #include <SDL2/SDL.h>
-#include <SDL2/SDL_mixer.h>
```

Comment:
This one's necessary - conftests are checking which headers are installed, and on Debian systems `libsdl2-mixer-dev` is a separate package.